spec/acceptance/zabbix_host_spec.rb

```diff
@@ -308,6 +340,63 @@ class { 'zabbix':
         end
 
       end
+
+      context 'test5.example.com' do
+        let(:test5) { result_hosts.select { |h| h['host'] == 'test5.example.com' }.first }
+
+        it 'is created' do
```

Comment:
I’ve duplicated the previously tested node, but should I skip to testing only `tls_accept`, `tls_connect`, `tls_issuer` and `tls_subject` ?

Reply:
I suppose some more attributes like `groups`, `macros` and `templates` are just spam and can be removed.
You can leave the basic test `is created`.

Reply:
done, thanks